DigitalExcellence/dex-frontend
# Fix accordion styling not being in the stylesheet of the overview component

## Description

I have moved the accordion styling to the overview component's stylesheet, and am targeting the bootstrap styled components by using the `:host` and `::ng-deep` pseudo-selectors.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Checkout this branch
2. Run the application
3. The accordion on the overview page should be styled

## Link to issue

Closes: #106
